inspirehep/modules/workflows/workflows/edit_article.py

```diff
@@ -40,6 +46,20 @@ def update_record(obj, eng):
     record = get_db_record('lit', control_number)
     record.update(obj.data)
     record.commit()
+    db.session.commit()
```

Comment:
actually there is https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/api.py#L272, it should be ok to remove it.